### DISCUSSION TITLE
Is it possible to patch fully colored svg images into a nerd font?

### DISCUSSION THREAD

**Discussion by ziontee113** [ryanoasis/nerd-fonts#870] [Q&A] [answered]
Hi, I wonder if it's possible patch a fully colored svg image into a nerd font to use in the terminal :)

My friend just tried and 1 image translated into 8 glyphs... Is this possible at all? Thank you :)

**Comment by Finii** [comment, marked as answer]
I have not much knowledge about color glyphs.

On my machine I have only one font with color glyphs, and that can not be opened by `fontforge`. `mate-font-viewer` does open it, though, and it is usable in `writer`. `showttf` and `ttfdump` show details. I doubt that any terminal emulator would be able to use it.

As we patch with `fontforge` I guess it is not possible. Maybe ask [there](https://github.com/fontforge/fontforge) if it is possible (and how) to even OPEN it.

Usually colors in terminals are not in the font but programmed in the shell via ANSI escape sequences on a per-glyph basis. But of course you know that and want a multicolored glyph.

_Later:_

Ah, this is the `fontforge` issue: https://github.com/fontforge/fontforge/issues/677
So the anser is 'not with `fontforge` and therefore not with `font-patcher`'.

**Reply by ziontee113** [reply]
Thank you very much :)

**Comment by ChiefMikeK** [comment]
FontForge can work with svg color layers  
the problem though is the different specs file formats between windoz mac google android firefox/Mozzilla...  
Termux displays andriod's emoji in color just fine.  others display hit or miss black/white  

https://fontforge.org/docs/ui/dialogs/multilayer.html

**Reply by Finii** [reply]
Just found this (by chance), do not know what it is worth, for investigation later.

https://glyphsapp.com/learn/creating-an-svg-color-font
